inspirehep/config.py

```diff
@@ -334,6 +338,74 @@
+    "773__y": ["publication_info.year"],
+    "anyfield": ["external_system_numbers.value",
+                 "external_system_numbers.obsolete",
+                 "report_numbers.value", "arxiv_eprints.categories",
+                 "languages", "authors.full_name", "authors.alternative_name",
+                 "authors.affiliations.value",
+                 "title_variation", "breadcrumb_title", "titles.title",
+                 "thesis.degree_type", "thesis.university",
+                 "thesis.date", "thesis_supervisor.affiliation",
+                 "public_notes.value", "abstracts.value",
+                 "free_keywords.value", "accelerator_experiments.experiment",
+                 "thesaurus_terms.keyword", "collaboration",
+                 "publication_info.recid", "publication_info.page_artid",
+                 "publication_info.journal_issue",
+                 "publication_info.conf_acronym",
+                 "publication_info.journal_title",
+                 "publication_info.reportnumber",
+                 "publication_info.confpaper_info",
+                 "publication_info.journal_volume", "publication_info.cnum",
+                 "publication_info.pubinfo_freetext", "publication_info.year",
+                 "publication_info.isbn", "publication_info.note",
+                 "isbns.value", "imprints.place", "imprints.publisher",
+                 "imprints.date", "thesis_supervisor.full_name",
+                 "title_translation.title", "title_translation.subtitle",
+                 "preprint_date", "corporate_author",
+                 "titles.subtitle", "subject_terms.term", "dois.value"
+                 ],
+    "authorcount": ["authors.full_name"],
+    "arXiv": ["arxiv_eprints.value"],
+    "caption": ["urls.description"],
+    "country": ["authors.affiliations.value"],
+    "firstauthor": ["authors.full_name", "authors.alternative_name"],
+    "fulltext": ["urls.url"],
+    "journal": ["publication_info.recid", "publication_info.page_artid",
+                "publication_info.journal_issue",
+                "publication_info.conf_acronym",
+                "publication_info.journal_title",
+                "publication_info.reportnumber",
+                "publication_info.confpaper_info",
+                "publication_info.journal_volume", "publication_info.cnum",
+                "publication_info.pubinfo_freetext", "publication_info.year",
+                "publication_info.isbn", "publication_info.note"
+                ],
+    "journal_page": ["publication_info.page_artid"],
+    "keyword": ["thesaurus_terms.keyword", "free_keywords.value"],
+    "note": ["public_notes.value"],
+    "reference": ["references.doi", "references.report_number",
+                  "references.journal_pubnote"
+                  ],
+    "subject": ["subject_terms.term"],
+    "texkey": ["external_system_numbers.value",
+               "external_system_numbers.obsolete"
+               ],
```

Comment:
Mmh. The texkey is really something precise. We should have a texkey field in the schema.